hadoop-plugin-test/src/main/gradle/positive/emergentFlow.gradle

```diff
@@ -0,0 +1,68 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+  dependencies {
+    classpath files("${project.pluginTestDir}/hadoop-plugin-${project.version}.jar", "${project.pluginTestDir}/hadoop-plugin-${project.version}-SNAPSHOT.jar")
+    // Must manually specify snakeyaml for testing, not required for users
+    classpath 'org.yaml:snakeyaml:1.18'
+  }
+}
+
+apply plugin: com.linkedin.gradle.hadoop.HadoopPlugin
+
+// Flow for testing yaml creation for emergent flows for Flow 2.0
+
+hadoop {
+  buildPath "jobs/emergentFlow"
+  cleanPath false
+
+  generateYamlOutput true
+
+  namespace('otherNamespace') {
+    propertyFile('emergentProperties') {
+      set properties: [
+              'new-parameter' : 'new-value',
+              'flow-level-parameter' : 'value2'
+      ]
+    }
+  }
+
+  workflow('emergentFlow') {
+    propertyFile('properties') {
+      set properties: [
+              'flow-level-parameter' : 'value'
+      ]
+    }
+
+    commandJob('shellBash') {
+      uses 'bash ./sample_script.sh'
+    }
+
+    commandJob('shellPwd') {
+      uses 'pwd'
+    }
+
+    commandJob('shellEcho') {
+      uses 'echo "This is an echoed text."'
+    }
+
+    targets 'shellPwd', 'shellEcho', 'shellBash'
+  }
+}
+
+hadoopZip {
+  libPath = "lib"
+  base {
+    from { fileTree("jobs/emergentFlow").exclude("otherNamespace").files } {
+      into "."
+    }
+  }
+  zip("emergentFlowTest") {
+    from { fileTree("jobs/emergentFlow/otherNamespace").files } {
+      into "."
+    }
+  }
+}
+
+startHadoopZips.dependsOn buildAzkabanFlows
```

Comment:
We usually have one workflow.gradle and one build.gradle file. In this test, it seems you combine them together in one single gradle file. Do they actually work in the same way?

Reply:
So I wasn't actually able to get this zip step to build - I think it's a consequence of gradle sequencing (i.e. it should be in the build.gradle file). I'd rather not add the layer of complication there, so I'll probably just remove this step. That means that this test is primarily used to confirm that the .tempprops file is created as expected, not necessarily to test the zipping step. I'm not sure of a good automated way to do this.